Installation/CHANGES.md

```diff
@@ -165,6 +165,8 @@ Release date: January 2022
 
 ### [Point Set Processing](https://doc.cgal.org/5.4/Manual/packages.html#PkgPointSetProcessing3)
 
+-   Added the function [`grid_simplify_point_set()`](https://doc.cgal.org/latest/Point_set_processing_3/group__PkgPointSetProcessing3Algorithms.html#gac7c81cc8a2986e3972e86612e4f847a1)
+    which merges points which belong to the same cell of a grid.
```

Comment:
As seems to be wrong: `grid_simplify_point_set` has been in CGAL since... at least CGAL-4.3: https://doc.cgal.org/4.3/Point_set_processing_3/group__PkgPointSetProcessing.html#gaaf22db6d8910092f66f045128706a60a.

Reply:
#5698 says "A new optional named parameter min_points_per_cell is added to grid_simplify_point_set(). The default value is 1 to keep the current behavior as default." under API changes.